course6/toyapp_thomasdhuys/drawlib/drawer.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include <vector>
+#include "primitives.h"
+#include "render_object.h"
+
+namespace Draw {
+
+class Drawer {
+public:
+	static RenderObject Draw();
+
+private:
+	static std::vector<IndexP> GetSquareIndices();
+	static RenderObject DrawSquarePoints();
+};
```

Comment:
I don't see a point in this object as well. It represents no concept, it's just a collection of functions